sparse_ho/models/logreg.py

```diff
@@ -380,15 +380,20 @@ def get_hessian(X, y, mask, dense, log_alpha):
             Logarithm of hyperparameter.
         """
         X_m = X[:, mask]
+        n_samples, size_supp = X_m.shape
         a = y * (X_m @ dense)
         temp = sigma(a) * (1 - sigma(a))
```

Comment:
I try to avoid variable names like this (when possible)

Reply:
`temp` >> `grad_sigmoid`